app/javascript/controllers/checkbox_filter_controller.js

```diff
@@ -0,0 +1,24 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  static targets = [ "input", "count", "countWrapper" ]
+  static classes = [ "displayable" ]
+
+  connect() {
+    this.updateCount()
+  }
+
+  updateCount() {
+    const length = this.inputTargets.filter(checkbox => checkbox.checked).length
+    this.setCountValue(length)
+  }
+
+  setCountValue(length) {
+    this.countTarget.innerHTML = length
+    if (length > 0) {
```

Comment:
We can use [toggle](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList):

```
this.countWrapperTarget.classList.toggle(this.displayableClass, length > 0)
```

Reply:
Oh cool, I didn't know toggle could take an argument like that.